### PR TITLE
support #EXTBYT/EXTIMG/EXTALB/EXTART/EXTGENRE and #EXTM3U attributes

### PR DESCRIPTION
**Pull request by freddy36**

### CODE REVIEW

**Comment by freddy36** [comment]
Can't see any comment, did you save it?

**Comment by Raiper34** [comment]
> Can't see any comment, did you save it?

Ahh sorry, I have started to review, but not completed it (not familiar with that functionality... ). Should be visible now.

File: src/m3u-playlist.ts [hunk 1]
```diff
@@ -29,7 +34,31 @@ export class M3uPlaylist {
...
-  urlTvg?: string = undefined;
+
+
+  /**
+   * Get url-tvg url
+   * @returns url-tvg url
+   * @deprecated The method should not be used, use playlist.attributes['url-tvg'] instead
+   */
+  get urlTvg(): string | undefined {
+    return this.attributes['url-tvg'];
```
[the remark this answers is not in the document]
Comment (freddy36) [reply src/m3u-playlist.ts:45]: fixed via https://github.com/Raiper34/m3u-parser-generator/pull/11/commits/4123a9429454f2783106ec1d9a7fa123ea669baa

**Comment by Raiper34** [comment]
Thanks! Will be released under v1.7.0 shortly these days.

**Pull request merged**